paasta_tools/tron_tools.py

```diff
@@ -650,6 +650,15 @@ def format_master_config(master_config, default_volumes, dockercfg_location):
         }
     )
     master_config["mesos_options"] = mesos_options
+
+    k8s_options = master_config.get("k8s_options", {})
+    if k8s_options:
+        # Only add default volumes if we already have k8s_options
+        # TODO: Gate this on whether k8s on tron is enabled
```

Comment:
i think that as long as k8s_options is there, we can go ahead and add these in

we can probably also just unconditionally (just like we do for mesos) do this once we've released a new tron version that won't barf on k8_options (which should be soon!)